common/src/main/java/com/netflix/conductor/common/utils/ExternalPayloadStorage.java

```diff
@@ -46,6 +46,22 @@ enum PayloadType {
      */
     ExternalStorageLocation getLocation(Operation operation, PayloadType payloadType, String path);
 
+    /**
+     * Obtain an uri used to store/access a json payload in external storage.
```

Comment:
In this line I would mention the deduplication based on payloadBytes digest

... also, mentioning md5 in the description of payloadBytes parameter should be removed because A) it is not true and B) it is an implementation detail